src/repository/dotRepository.js

```diff
@@ -13,19 +19,40 @@ async function getId(scriptId, actorId) {
   return result.rows[0].id;
 }
 
+/**
+ * Update the position of the progress dot
+ * @param {number} dotId ID of the current progress dot
```

Comment:
nit:  
"How do I know what the _current_ progress dot is?" - some person    

Similar to my previous comment, the extra context here implies that there is some extra consideration that needs to be taken before passing an id.  
Like, I need to get the "current" progress dot id from elsewhere because if I pass an Id that isn't "current" then something might go wrong.  
  
In this case, it doesn't matter which progress dot the ID corresponds to, the function will work with any.